### ISSUE TITLE
Notify about new feedback

### ISSUE THREAD

**Issue by dtu-compute** [dtu-compute/dtu-enote-website#217]
It si easy for the professor to forget to check for new feedback.

Perhaps there should be something like this

```
Notify by email

[]  Iain Bryson
[x] Karsten Schmidt
[]  Martin Jeppesen
```

in Admin->Feedback, which is a list the admins for that course.

**Comment by iainbryson** [comment]
Note that adding this kind of thing could likely mean adding a jobs server (e.g. Sidekiq) to queue and execute the emails.  It's not a huge deal, but it does mean a new container.

**Comment by dtu-compute** [comment]
Very good to know. Maybe a modal up or similar after logging in, if there are feedback without an admin comment. Or maybe a dashboard type page right after login. Would prefer not introducing another container =)

**Comment by iainbryson** [comment]
Yes, or maybe one of those (N) red circle badges on the admin menu bar that would tell the admin that there are new feedbacks to review

**Comment by dtu-compute** [comment]
That would be the most preferred for sure =)

**Comment by iainbryson** [comment]
Why is this phase 0 too?

**Comment by dtu-compute** [comment]
If anything it should be a bug fix, but that doesn't make sense either. I'll remove the phase 0 label.
